Nesting Navigators (#6)
neocoaster/react-native-boilerplate: README.md

```

This is a component that manages the navigation tree and contains the navigation state. It provides two important props: [`navigation`](https://reactnavigation.org/docs/navigation-prop/) and [`route`](https://reactnavigation.org/docs/route-prop/) to your screens. The first one, is generally used for navigation between screens, going back, making changes to the route's params, etc. The `route` prop is used to pass params between screens and get their name.

***Nesting Navigators***

In the branch `nesting-navigators` you can find an example of a tab navigator nested inside a stack navigator:

- Stack.Navigator
  - Screen 1 (Screen)
  - Screen 2 (Screen)
  - Screen 3 (Screen)

>   - Dashoboard (Tab.Navigator)

Reviewer: Typo in `Dashboard`

    - Account (Screen)
    - Files
    - Settings

> You can find the code for creating the new Tab Navigator on the `navigation/DashboardTabNavigator.js`, here we config three new screens for the Dashboard: `Account`, `Files` and `Settings`. The order we put them on the code, is the order it will show on the app left to rigth.

Reviewer: ```suggestion
You can find the code for creating the new Tab Navigator on the `navigation/DashboardTabNavigator.js`, here we config three new screens for the Dashboard: `Account`, `Files` and `Settings`. The screens will show up on the app starting on the left in the same order that they appear in the code.
```

> You can find the code for creating the new Tab Navigator on the `navigation/DashboardTabNavigator.js`, here we config three new screens for the Dashboard: `Account`, `Files` and `Settings`. The order we put them on the code, is the order it will show on the app left to rigth.

Reviewer 2: `configure` instead of `config`? @lilhand

Reviewer: I like 'config' more than `configure` honestly, but since you point it out, I guess you're not really configuring anything there, instead I'd just use 'create'.

> For navigate to a screen in a nested navigator we do:

Reviewer: ```suggestion
To navigate to a screen in a nested navigator we use:
```

```
navigation.navigate('Dashboard');
```

> And the initial screen inside the `Dashboard` component is show, which is `Account` in this case. If you want to navigate to a specific screen, you pass the name of the screen in params:

Reviewer: ```suggestion
And the initial screen inside the `Dashboard` component is shown, in this case `Account`. To navigate to a specific screen, you pass the name of the screen in params:
```

```
navigation.navigate('Dashboard', { screen: 'Files' });
```

Now, the `Files` screen will be rendered instead of `Account` upon navigation.

> You can follow similar approach for deeply nested screens:

Reviewer: ```suggestion
You can follow a similar approach for deeply nested screens:
```
